package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "nodemon ./server/index.js",
-    "build": "webpack -d --watch"
+    "build": "webpack -d --watch",
```

Comment:
should your `build` command trigger a `--watch`? what is `build` intended to do?

Reply:
To piggyback on what Nick said - I think usually build is for building the production version. I use start for my dev version of webpack. You also will probably/possibly (prossibly?) want a webpack config file and you'll call that here too.  :D